### PR TITLE
Check timeouts

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Add default timeout for each check via constant.
- Check can override the timeout.

---

- [ ] Override the default value via CLI/colin.run()
    - Override also the per-check timeout? Two options?

### CODE REVIEW

File: colin/utils/cmd_tools.py [hunk 1]
```diff
@@ -85,3 +93,30 @@ def is_rpm_installed():
     except FileNotFoundError:
         rpm_installed = False
     return rpm_installed
+
+
+def exit_after(s):
```
Comment (TomasTomecek) [inline colin/utils/cmd_tools.py]: I just realized you could have use Probe from conu. But since it's done already, let's use your implementation.

**Comment by lachmanfrantisek** [comment]
> A test for the new decorator wouldn't hurt.

Agree, I'll try to create some.

File: colin/utils/cmd_tools.py (hunk 1 above)
Comment (lachmanfrantisek) [reply colin/utils/cmd_tools.py]: Thanks for pointing it up -- I don't know about that.

**Comment by TomasTomecek** [comment]
Let me know when you feel like we can merge this.

**Comment by lachmanfrantisek** [comment]
@TomasTomecek I've added some tests and I feel like we can merge this for now..;-)

File: colin/core/constant.py [hunk 2]
```diff
@@ -35,3 +35,5 @@
 }
 
 COLIN_CHECKS_PATH = "CHECKS_PATH"
+
+CHECK_TIMEOUT = 10 * 60  # s
```
Comment (lachmanfrantisek) [inline colin/core/constant.py]: Just one question -- is the default timeout ok?

**Pull request merged**